e2e-tests/tests/test_e2e_integration_flows.py

```diff
@@ -200,11 +200,16 @@ def test_demo_website_text_returns_correct_anonymized_version():
     text = " ".join(text_into_rows)
     language = "en"
     score_threshold = 0.35
+    entities = ["PHONE_NUMBER", "US_DRIVER_LICENSE", "US_PASSPORT",
```

Comment:
This test makes sure the demo website's text is anonymized correctly. While this test would pass by selecting specific entities, it would not guarantee that the demo website would work properly (as it uses all the default entities). 
Two possible solutions I can think of:
1. Make sure the new entities don't change the demo website's result by changing the logic for AU entities to avoid false positives
2. Alter the demo website's text to make sure there are no false positives. If you want to go this path we can sync offline and think together about that.
3. Have the new AU entities not part of the default presidio installation. 
 
False positives, especially for new users trying out the service, are considered bad experience, although we know that these are difficult to avoid on a real system.

Reply:
Thanks @omri374 for the feedback. Yes, that makes sense, too many false positives would be a bad user experience. I think changing the score for the weak regex in the other comments will help resolve this (since they will be below the 0.35 score threshold). I think should resolve it because the new entities have quite generic patterns, but the checksum helps distinguish them quite well.

Reply:
Hi, update on the above. Changing the scores seems to have resolved the issue except for the "passport" test case. It clashes with "AU_TFN" because it just so happens that it is a valid TFN (big coincidence). This could be resolved by changing the test passport number by one digit (given passport does not have a checksum validation). Please let me know your thoughts.

Reply:
Sounds good. Let's go ahead with it. I'll just need the new passport number to update the demo website's text before merging this one.